tests/components/application-connector/resources/charts/gateway-test/templates/target-api-mock/mock-app.yml

```diff
@@ -2,31 +2,49 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: mock-application
-  name: mock-application
+    app: {{ .Values.mockServiceName}}
```

Comment:
I'm not sure if we need to parametrize it. We have this name hardcoded in the Makefile anyway. What do you think? I'd remove it from the `values.yaml`